test/box-luatest/gh_6305_autocomplete_metamethod_test.lua

```diff
@@ -0,0 +1,54 @@
+local t = require('luatest')
```

Comment:
You don't need box for this test so please move this to the `app-luatest` suite.

Reply:
moved